cypress/e2e/addPerson.feature

```diff
@@ -146,9 +146,6 @@ Feature: Add a new person to a tenure
     And I click edit person
     And I click cancel
     Then the confirmation modal is displayed
-  @Accessibility
-  Scenario: Accessibility Testing
-    And have no detectable a11y violations
```

Comment:
This test has been removed from here:
1. it's incomplete
2. the same test is set up in the `google-lighthouse.js` file (for this specific page)